src/appleseed-max-impl/appleseedlightmtl/appleseedlightmtl.cpp

```diff
@@ -149,6 +150,12 @@ namespace
             p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_EDIT_LIGHT_POWER, IDC_SPINNER_LIGHT_POWER, SPIN_AUTOSCALE,
         p_end,
 
+        ParamIdLightExposure, L"light_exposure", TYPE_FLOAT, P_ANIMATABLE, IDS_LIGHT_EXPOSURE,
+            p_default, 0.0f,
+            p_range, 0.0f, 100.0f,
```

Comment:
The range should allow negative values. We usually use [-64, 64] as exposure range.